packages/@webex/internal-plugin-voicea/package.json

```diff
@@ -41,7 +41,7 @@
     "sinon": "^9.2.4"
   },
   "scripts": {
-    "build": "yarn build:src",
+    "build": "yarn run -T tsc --declaration true --declarationDir ./dist",
```

Comment:
Removing this for now. As testing this will require testing in web-client. We have a similar JIRA to fix types in plugin-meetings. Updated that JIRA to test and fix this as well.
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-502998